spk/nzbget-testing/src/service-setup.sh

```diff
@@ -1,9 +1,9 @@
 PATH="${SYNOPKG_PKGDEST}/bin:${PATH}"
 NZBGET="${SYNOPKG_PKGDEST}/bin/nzbget"
-CFG_FILE="${SYNOPKG_PKGDEST}/var/nzbget.conf"
+CFG_FILE="${SYNOPKG_PKGVAR}/nzbget.conf"
```

Comment:
Reviewing this, I must say I really much like the `${SYNOPKG_PKGVAR}`.
But while at it, why not add a `${SYNOPKG_PKGBIN}` and `${SYNOPKG_PKGSHARE}`?

Reply:
This is a newly set variable from DSM7 and points to a different folder:

```
SYNOPKG_PKGVAR: /var/packages/{package}/var
SYNOPKG_PKGDEST/var: /var/packages/{package}/target/var
```
On pre DSM7, `SYNOPKG_PKGVAR` gets set to `SYNOPKG_PKGDEST/var` to keep backwards compatibility and make migrating packages easier. Adding our own I would use a different prefix: `SC_PKG_PKGSHARE` what use cases do you have for those variables? Or what should they be set to?

In any case I think this is out of scope at the moment.

Why do any of it? On dsm7 `SYNOPKG_PKGDEST` gets overridden during package upgrades. So `SYNOPKG_PKGDEST/var` can't store settings anymore, see the wiki for the new folder structure.

Reply:
While looking at the code `SYNOPKG_PKGDEST/bin` and `SYNOPKG_PKGDEST/share` came up quite often where you have modified the `var` ones.  I though that for consistency it may be worth while.  Out of scope, indeed, but best window of opportunity for such a change is probably now :)

Reply:
Thanks, I understand, but  since we agree it's out of scope I'll leave it for now. Personally I think keeping `SYNOPKG_PKGDEST/bin` makes the intent clearer than hiding it behind another variable. I actually hope some packages can make use of the new variables set by synology. e.g : `SYNOPKG_PKGHOME`, `SYNOPKG_PKGDEST_VOL` `SYNOPKG_TEMP_UPGRADE_FOLDER`